db/schema.rb

```diff
@@ -195,6 +195,9 @@
     t.string "domain"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.string "full_name"
+    t.string "site_domain"
```

Comment:
i'm gunna forget what this is - can we add [column comments](https://www.postgresql.org/docs/current/sql-comment.html)?